cmd/ssh-tpm-keygen/main.go

```diff
@@ -64,23 +64,23 @@ Example:
     $ ssh-tpm-keygen
     Generating a sealed public/private ecdsa key pair.
     Enter file in which to save the key (/home/user/.ssh/id_ecdsa):
-    Enter pin (empty for no pin):
-    Enter same pin again:
+    Enter passphrase (empty for no passphrase):
+    Enter same passphrase again:
     Your identification has been saved in /home/user/.ssh/id_ecdsa.tpm
     Your public key has been saved in /home/user/.ssh/id_ecdsa.pub
     The key fingerprint is:
     SHA256:NCMJJ2La+q5tGcngQUQvEOJP3gPH8bMP98wJOEMV564
     The key's randomart image is the color of television, tuned to a dead channel.`
 
-func getPin() []byte {
+func getPassphrase() []byte {
 	for {
-		pin1 := askpass.ReadPassphrase("Enter pin (empty for no pin): ", askpass.RP_ALLOW_STDIN|askpass.RP_NEWLINE)
-		pin2 := askpass.ReadPassphrase("Confirm pin: ", askpass.RP_ALLOW_STDIN|askpass.RP_NEWLINE)
-		if !bytes.Equal(pin1, pin2) {
+		pass1 := askpass.ReadPassphrase("Enter passphrase (empty for no passphrase): ", askpass.RP_ALLOW_STDIN|askpass.RP_NEWLINE)
+		pass2 := askpass.ReadPassphrase("Confirm passphrase: ", askpass.RP_ALLOW_STDIN|askpass.RP_NEWLINE)
+		if !bytes.Equal(pass1, pass2) {
 			fmt.Println("Passphrases do not match.  Try again.")
```

Comment:
That this used `Passphrase` already, but `Enter pin` previously hints we should opt for one-or-other probably